src/jupyter_contrib_nbextensions/nbextensions/2to3_cell/2to3_cell.yaml

```diff
@@ -0,0 +1,11 @@
+Type: IPython Notebook Extension
+Name: 2to3_cell
+Description: Converts python2 code in a notebook's code cell to python3 code
+Link: README.md
+Main: 2to3_cell.js
+Compatibility: Jupyter (4.x)
+Parameters:
+- name: conv2to3.hotkey 
+  description: Hotkey to convert the current cell from python2 to python3.
+  input_type: text
```

Comment:
You can use `input_type: hotkey` to get the configurator to provide a key-binding editor for you (see for example [toggle_all_line_numbers/main.yaml#L11](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/blob/3f1122c28723e4f726596dfc153c1976c29339e6/src/jupyter_contrib_nbextensions/nbextensions/toggle_all_line_numbers/main.yaml#L11))